HISTORY.rst

```diff
@@ -2,6 +2,22 @@
 History
 =======
 
+2.1.0b1 (2018-10-05)
+--------------------
+
+* It's happening! Hot on the heels of the last release, CumulusCI is making the jump to the modern era by adding **support for Python 3**! (Specifically, Python 3.6 and 3.7.) Don't worry, we'll also continue to support Python 2 until it reaches end-of-life at the start of 2020. Because this is a bit more wide-reaching change than normal, we're releasing a beta first. To install the beta you'll need to explicitly request its version: ``pip install cumulusci==2.1.0b1``.
```

Comment:
(Specifically, 3.6+.) ?

Reply:
Is there a way to `pip install cumulusci>=2.1.0b1` (or something) and the `pip install -U cumulusci` keeps you on the beta track? probably not but...